### PR TITLE
NXP-26687: better WARN message on S3 copy failure

### PR DESCRIPTION
**Pull request by efge**

### CODE REVIEW

File: nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3BinaryManager.java [hunk 1]
```diff
@@ -529,7 +530,10 @@ protected String copyBlob(S3BinaryManager sourceBlobProvider, String blobKey) th
             }
             return digest;
         } catch (AmazonServiceException e) {
-            log.warn("direct S3 copy not supported, please check your keys and policies", e);
+            String message = "S3 copy not supported from s3://" + sourceBucketName + "/" + sourceKey + " to s3://"
+                    + bucketName + "/" + key + " (" + length + " bytes)";
+            log.warn(message + ", falling back to regular copy: " + e.getMessage());
```
Comment (kevinleturc) [inline nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3BinaryManager.java]: Is it on purpose to not give exception to warn ?

**Comment by kevinleturc** [comment]
👍

**Comment by funshodavid** [comment]
:+1:

File: nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3BinaryManager.java (hunk 1 above)
Comment (efge) [reply nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3BinaryManager.java]: Yes, as the full stack trace doesn't give anything very meaningful and we can get it in debug.

**Pull request merged**